<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="searchListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>800</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>下载共享文件</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>641</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>30</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QListWidget" name="responseListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>99</y>
        <width>780</width>
        <height>431</height>
       </rect>
      </property>
      <property name="gridSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>请输入搜索关键词：（Note：大小写敏感）</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索结果：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>下载文件：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>540</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>下载进度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>540</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>源IP：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>文件大小:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>源IP:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>上传共享文件</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_addFile">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>添加文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addDir">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>添加目录</string>
      </property>
     </widget>
     <widget class="QListWidget" name="sharingListWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>761</width>
        <height>501</height>
       </rect>
      </property>
      <property name="gridSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>加入文件/目录：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="loadListWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>590</y>
      <width>791</width>
      <height>131</height>
     </rect>
    </property>
    <property name="gridSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
